microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -53,7 +53,7 @@ These features are described below.
 <!-- ====================================================================== -->
 ## SmartScreen
 
-Microsoft Defender SmartScreen ("SmartScreen") is a security feature that is enabled by default to help users safely browse the web.  The `IsReputationCheckingRequired` property controls whether SmartScreen is enabled.  Generally, all other services in `edge://settings/privacy` are turned off, for WebView2.
+Microsoft Defender SmartScreen ("SmartScreen") is a security feature that is enabled by default to help users safely browse the web.  The `IsReputationCheckingRequired` property controls whether SmartScreen is enabled.  Generally, all other services in Microsoft Edge > **Settings and more** (![Settings and more icon](./data-privacy-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security** (and in **Settings and more** (![Settings and more icon](./data-privacy-images/settings-and-more-icon.png)) > **Settings** > **Search and services**) are turned off, for WebView2.

```

Comment:
SmartScreen - review this change - delete the part about "Search and services"?
@PatrizioBytes @apryzgoda

Reply:
I think it's fair to retain "(and in ... Search and services"), since that's implied by the original wording.

Reply:
result: A.P. offline: "LGTM"